byway/src/main/webapp/routepage.js

```diff
@@ -292,5 +292,34 @@ function createRecButton(rec){
   return recBtn;
 }
 
-/* exported initMap, generateRoute, placesService, map */
-/* global google, findPlace, getTripKeyFromUrl, configureTripKeyForPath */
\ No newline at end of file
+
+/**  Creates a URL link to Google Maps based on the start/end point and route
+ *   @returns {String} routeLink url containing query params for the user’s route
+ */
+function generateRouteLink(){
+    let routeLink = "https://www.google.com/maps/dir/?api=1&travelmode=driving" 
+    routeLink += "&origin=" + start.name.split(' ').join('%20') + "&destination=" + end.name.split(' ').join('%20') +"&waypoints=";
+    for(let place of route){
+      console.log(place);
+      routeLink += place.name.split(' ').join('%20') + "|";
+    }
+   return routeLink;
+}
+
+
+function sendEmail(){
+  fetch('/api/email')
+  .then(response => response.json())
+  .then(email =>{ 
+    let emailLink = "mailto:" + email + "?subject=Your%20Roadtrip%20Plan&body=Thank%20you%20for%20using%20Byway!%20" +
+      "Your%20destinations%20are%20listed%20below.%20Click%20the%20link%20to%20see%20your%20route%20in%20Google%20Maps.%0D%0A" 
```

Comment:
small nit: this says you destinations are listed below, so I think we should either list the destinations as well or take out that sentence

Reply:
I just said "route" to be more general, thanks for pointing it out!